src/Components/NewComment.jsx

```diff
@@ -0,0 +1,35 @@
+import { useState, useEffect } from "react";
+import { postAComment } from "../assets/Requests/api";
+
+const NewComment = ({ article_id, setComments }) => {
+  const [comment, setComment] = useState("");
+  const [username, setUsername] = useState("jessjelly");
+
+  useEffect(() => {
+    setUsername("jessjelly");
+  }, []);
+
+  const handleSubmit = (event) => {
+    event.preventDefault();
+    postAComment(article_id, username, comment).then((newComment) => {
+      setComments((currComments) => {
+        return [newComment, ...currComments];
+      });
+    });
+  };
+
+  return (
+    <form onSubmit={handleSubmit}>
+      <label htmlFor="comment">Comment:</label>
+      <textarea
+        id="comment"
+        value={comment}
+        onChange={(event) => {
+          setComment(event.target.value);
```

Comment:
could do with some form validation. Currently a user can post an empty comment and this is leading to an axios error in the terminal, but a blank comment is still added.  Think about how you can prevent this before any API query is made.